igvm/cli.py

```diff
@@ -177,6 +177,12 @@ def parse_args():
             ' operator to shut down VM.'
         ),
     )
+    subparser.add_argument(
+        '--enforce-env',
+        dest='enforce_env',
+        action='store_true',
+        help='Build or migrate VM only to a HV with the same environment'
```

Comment:
Maybe write same environment of VM to avoid miss understandings.

Reply:
added